lib/msf/base/simple/auxiliary.rb

```diff
@@ -68,21 +68,25 @@ def self.run_simple(omod, opts = {}, &block)
       mod.init_ui(nil, nil)
     end
 
-    ctx = [ mod ]
+    run_uuid = Rex::Text.rand_text_alphanumeric(24)
```

Comment:
could we rename run_uuid or make it store a real UUID instead? 
I could see this causing confusion if someone (me) was trying to debug wondering why the uuid isn't a uuid :sweat_smile:

Reply:
I largely agree, however Metasploit has a long history of calling things "UUID" that don't meet any of the UUID specs (influenced by a number of things, I imagine, including the historical [`Object#id`](https://ruby-doc.org/core-1.8.7/Object.html#method-i-id) complicating calling anything just `id` and RFC 4122 passing in the year this codebase started). This value is taking the place of `module.uuid` (only 8 alnum bytes!) in the RPC API https://github.com/rapid7/metasploit-framework/blob/master/lib/msf/core/rpc/v10/rpc_module.rb#L671. Then there's [payload UUIDs](https://github.com/rapid7/metasploit-framework/blob/master/lib/msf/core/payload/uuid.rb). Fixing the situation would be nice, but will take some thought and is outside the scope of this PR.